spring-integration-ip/src/main/resources/org/springframework/integration/ip/config/spring-integration-ip-4.3.xsd

```diff
@@ -80,8 +80,17 @@
 							</xsd:appinfo>
 						</xsd:annotation>
 					</xsd:attribute>
-					<xsd:attribute name="host" type="xsd:string" />
+					<xsd:attribute name="host" type="xsd:string" use="optional"/>
 					<xsd:attribute name="acknowledge" type="xsd:string" />
+					<xsd:attribute name="socket-expression" type="xsd:string" use="optional">
+						<xsd:annotation>
+							<xsd:documentation>
+								An SpEL expression to be evaluated to determine which datagram
```

Comment:
"A SpEL...."
The article `an` is used when the target word starts from vowel :smile: 

And "socket **to** use" in the next line.